docs/source/install.rst

```diff
@@ -70,18 +76,46 @@ And then compiling using CMake:
     cd ROSCO
     mkdir build
     cd build
-    cmake ..                        # Mac/Linus
-    cmake .. -G "MinGW Makefiles"   # Windows
+    cmake ..
     make install
 
-This will generate a file called :code:`libdiscon.*` in the :code:`/ROSCO/install/lib` directory. 
+This will generate a file called :code:`libdiscon.so` in the :code:`/ROSCO/install/lib` directory. 
```

Comment:
Point out that the library extension is `.dylib` for mac?

Reply:
Done.